client/src/components/pages/Settings.jsx

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+import classes from './../../styles/Settings.module.css';
+
+const Set = () => {
```

Comment:
Лучше не сокращать имена
```suggestion
const Settings = () => {
```